src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
`element.localName` will pass pretty much no matter what, even if there's no registered custom element with that name. E.g. paste this into your devtools on this very page: `document.createElement('foo-bar').localName`

The Polymer telemetry assert is a little crazy, that's fair. What about just the other two?

```javascript
// Test that the element was registered properly.
assert.true(!!customElements.get('<%= elementName %>'));
assert.true(element instanceof customElements.get('<%= elementName %>'));
```

That'll end up looking like:

```javascript
// Test that the element was registered properly.
assert.true(!!customElements.get('my-element'));
assert.true(element instanceof customElements.get('my-element'));
```